Lykke.Job.TradelogBridge.Azure/TradesRepository.cs

```diff
@@ -25,18 +26,27 @@ public async Task InsertTradesAsync(List<Trade> trades)
         {
             _log.Info("Inserting Trades into Table Storage");
 
-            try
+            foreach (var trade in trades)
             {
-                var groupedTrades = trades.GroupBy(t => t.PartitionKey);
-                foreach (var group in groupedTrades)
+                try
                 {
-                    await _tradesTableStorage.InsertAsync(group);
+                    await _tradesTableStorage.InsertAsync(trade);
+                }
+                catch (StorageException storageEx)
+                {
+                    if (storageEx.RequestInformation.HttpStatusCode == (int)HttpStatusCode.Conflict)
+                    {
+                        _log.Warning("Duplicated Trade row found", null, trade);
+                        continue;
+                    }
+                    _log.Error("Error inserting Trade into Table Storage", storageEx, storageEx.Message, trade);
```

Comment:
for log.Error it's better to use overload (Exception ex, string message, object context,..), so your invocation should look like:
```
_log.Error(storageEx, "Error inserting Trade into Table Storage", trade);
```

the same for other error messages below. This invocations logs containing class name and method name as `Component` and `Process` log fields respectively, which is common practice in Lykke UK.